docroot/profiles/custom/sitenow/sitenow.install

```diff
@@ -4186,3 +4187,32 @@ function sitenow_update_10024() {
       ->save();
   }
 }
+
+/**
+ * Rebuild search index for sitenow_intranet sites.
+ */
+function sitenow_update_10025() {
+  /** @var Drupal\Core\Config\ImmutableConfig $config */
+  $config = \Drupal::service('config_split.manager')->getSplitConfig('config_split.config_split.sitenow_intranet');
+
+  // If intranet split isn't registered or
+  // isn't active, return early.
+  if (!isset($config) || !$config->get('status')) {
+    return t('Intranet split is not active, so no update to run.');
+  }
```

Comment:
Could this be moved to the `sitenow_intranet.install` and this check be removed instead?